source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -394,7 +394,11 @@ def _get_name(self):
 				textList.append("%s: %s" % (header, content))
 			else:
 				textList.append(content)
-		return "; ".join(textList)
+		name = "; ".join(textList)
+		#Some list view items in Vista and later can contain annoying left-to-right and right-to-left indicator characters which really should not be there.
+		name=name.replace(u'\u200E','')
+		name=name.replace(u'\u200F','')

```

Comment:
Hmmm, in one place, a one liner is used, whereas this and other lines use two liners. I think this isn't quite consistent when people read the code, so I'd like to suggest using a consistent style. Thanks.

Reply:
Hmm, is there a particular reason why you didn't address this yet?